test/fixtures/transform_params.html

```diff
@@ -0,0 +1,3 @@
+<span>{{ content.string | repeat:4 }}</span>
+<span>{{ content.string | replace:'B':'C' }}</span>
```

Comment:
This is pretty nice! Not entirely sure how I feel about the syntax. How about those:
```
replace('B', 'C')
replace:'B','C'
```
I think I'm just picky about the second colon `:`. I'm fine with the seperation between name and arguments with a colon, but I think the next argument should be separated by commas. Might make the parsing slightly more annoying, right?